src/pages/scientificServices/pipelines/components/ImputationPrivatePreviewGate.tsx

```diff
@@ -0,0 +1,64 @@
+import { ButtonPrimary, Icon } from '@terra-ui-packages/components';
+import React, { ReactNode } from 'react';
+import colors from 'src/libs/colors';
+import { isFeaturePreviewEnabled } from 'src/libs/feature-previews';
+import { IMPUTATION_UI } from 'src/libs/feature-previews-config';
+import { SCIENTIFIC_SERVICES_SUPPORT_EMAIL } from 'src/pages/scientificServices/pipelines/common/scientific-services-common';
+
+interface FeaturePreviewLandingProps {
+  children: ReactNode;
+}
+
+export const ImputationPrivatePreviewGate = ({ children }: FeaturePreviewLandingProps): ReactNode => {
+  const isFeatureEnabled = isFeaturePreviewEnabled(IMPUTATION_UI);
+  return isFeatureEnabled ? (
+    children
+  ) : (
+    <div
+      style={{
+        margin: '4rem 3rem',
+        padding: '1rem',
+        width: '40%',
+        minWidth: '300px',
+      }}
+    >
+      <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', marginBottom: '1rem' }}>
+        <Icon icon='info-circle-regular' size={32} style={{ color: colors.primary(), marginRight: '0.5rem' }} />
+        <h2>
+          Thank you for registering for the{' '}
+          <span
+            style={{
+              fontStyle: 'italic',
+            }}
+          >
+            All of Us
+          </span>{' '}
+          + AnVIL Imputation Service.{' '}
+        </h2>
+      </div>
+      <div style={{ marginLeft: '2.5rem', marginTop: '1rem' }}>
+        The All of Us + Anvil Imputation Service user interface is currently in private preview and is not yet available
```

Comment:
i'll get product feedback on this text as well before merging.

Reply:
good callout, at the very least they're gonna want to change `Anvil` -> `AnVIL`